src/fake-xml-http-request.js

```diff
@@ -381,8 +380,8 @@ var FakeXMLHttpRequestProto = {
 
     if (this.readyState == FakeXMLHttpRequest.DONE) {
       this.dispatchEvent(new _Event("load", false, false, this));
-      this.dispatchEvent(new _Event("loadend", false, false, this));
     }
+    this.dispatchEvent(new _Event("loadend", false, false, this));
```

Comment:
This is actually bug now, because every time  _readyStateChange is called this "loadend" gets dispatched. For example FakeXMLHttpRequest.OPEN is dispatched and follows with `loadend`.